src/index.html

```diff
@@ -28,6 +28,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/img/global/icons/favicon-32x32.png" >
     <link rel="icon" type="image/png" sizes="16x16" href="/img/global/icons/favicon-16x16.png" >
     <link rel="icon" sizes="48x48" href="/img/global/icons/favicon.ico" >
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
We don't need to import this stylesheet, we already already have the `@material-ui/icons` package installed.

Reply:
Good to know, I will exclude the stylesheet, thanks @lazorfuzz